packages/relay/src/lib/eth.ts

```diff
@@ -1542,6 +1613,16 @@ export class EthImpl implements Eth {
     return logs;
   }
 
+  private async getAccountLatestEthereumNonce(address: string, requestId?: string) {
```

Comment:
Appropriately named. Potentially though this doesn't work when account is a contract since contract nonce is likely not externalized yet